exchanges/coinbasepro/coinbasepro_types.go

```diff
@@ -365,29 +365,34 @@ type ErrorResponse struct {
 	NewOrderFailureReason string `json:"new_order_failure_reason"`
 }
 
+// SharedOrderConfig contains order configuration information used in both PlaceOrderInfo and PreviewOrderInfo
+type SharedOrderConfig struct {
```

Comment:
🦍  I don't think we need to say it's shared. It's just `OrderConfig`.
I think you're only thinking about it being shared because of where you started.
```suggestion
type OrderConfig struct {
```

Do not need to action.

Reply:
The OrderConfiguration naming comes from what the exchange itself calls it. Would like to double-check changing it.

But I'll apply that change from SharedOrderConfig > OrderInfo.